bundles/org.openhab.binding.sbus/src/main/resources/OH-INF/thing/bridge-udp.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="udp">
+		<label>Sbus UDP Slave</label>
```

Comment:
Not sure if the term slave is appropiate, as it is a bridge, not sure if every reader understands this.
```suggestion
		<label>SBUS UDP Bridge</label>
```